launch/fluentd-kinesis-forwarder-monitor.yml

```diff
@@ -0,0 +1,14 @@
+run:
+  type: docker
+env:
+- ENV_SCOPE
+- LOG_FILE # TODO: remove
```

Comment:
Ahh thanks for the reminder.  Mind also reviewing this PR: https://github.com/Clever/ark-config/pull/422